generate_rospkg_apkbuild/genapkbuild.py

```diff
@@ -283,23 +312,35 @@ def package_to_apkbuild(ros_distro, package_name,
     if ver_suffix is None:
         ver_suffix = ''
 
+    url = ""
+    if len(pkg.urls) > 0:
+        url = pkg.urls[0].url
+    else:
+        if not is_ros2:
+            url = 'http://wiki.ros.org/$_pkgname'
+        else:
+            url = 'https://index.ros.org/p/$_pkgname'
+
     g = {
         'pkgname': ros_pkgname_to_pkgname(ros_distro, pkg.name),
         '_pkgname': pkg.name,
         'pkgver': pkg.version + ver_suffix,
         'pkgrel': revfn(pkg.version + ver_suffix),
         'ros_distro': ros_distro,
-        'url': pkg.urls[0].url if len(pkg.urls) > 0 else 'http://wiki.ros.org/$_pkgname',
+        'url': url,
         'license': pkg.licenses[0],
         'check': check,
-        'depends': depends_keys + depends_export_keys,
-        'makedepends': makedepends_implicit + makedepends_keys,
+        'depends': sorted(list(set(depends_keys + depends_export_keys))),
+        'makedepends': sorted(list(set(makedepends_implicit + makedepends_keys))),
```

Comment:
Could you separate the PR? This will also updates all ros1 packages

Reply:
Done in https://github.com/alpine-ros/ros-abuild-docker/pull/158/commits/a9db76f833716b22f6abe4633da471a36de27b18